app/views/api/v1/stat_mailer/send_stat_email.text.erb

```diff
@@ -0,0 +1,9 @@
+Your Brain Freeze Statistics Report
+The group average correct answers: <%= @stat.avg_correct_answers %>
+Topic: <%= @stat.game.topic %>
+Number of questions: <%= @stat.game.number_of_questions %>
+
+<% @stat.game.players.each do |player| %>
+  Player Name: <%= player.display_name %>
+  Player average correct answers: <%= player.answers_correct %>
```

Comment:
Similar suggestion unless we want to have it calculate the player's average correct answers
```
<p>Player's total correct answers: <%= player.answers_correct %></p>
```

Reply:
Perfect, thank you for pointing that out, is included in the latest push